### ISSUE TITLE
FEATURE: Create way of showing closest places with table (for people who can't read map)

### ISSUE THREAD

**Issue by raven-wing** [Problematy/goodmap#107]
**Is your feature request related to a problem? Please describe.**
Map is difficult to use for people with special needs.

**Describe the solution you'd like**
There should be accessibility button which would change map for list of places, sorted by distance from provided location

**Describe alternatives you've considered**
It could be one address with button 'navigate me'
